documentation/staging/content/managing-domains/accessing-the-domain/istio/istio.md

```diff
@@ -52,6 +56,8 @@ The current support for Istio has these limitations:
   and has been tested with single and multicluster
   Istio installations from 1.10 up to 1.13.2.
 
+* For Red Hat OpenShift Service Mesh, continue to use Operator version 3.4.  This version of Operator is incompatible with OpenShift Service Mesh base Istio version implementation which is 1.9.
```

Comment:
Operator -> operator (everywhere unless it is in the product name WebLogic Kubernetes Operator)
This version of Operator -> This version of the operator
Add commas -> OpenShift Service Mesh, base Istio version implementation, which is 1.9.